k830 support + support for NEW FN INVERSION

decibyte opened the pull request.

Lekensteyn commented on lib/logitech_receiver/descriptors.py:
```diff
@@ -191,6 +191,13 @@ def _D(name, codename=None, kind=None, wpid=None, protocol=None, registers=None,
 							_RS.hand_detection(),
 						],
 				)
+#'''
```
Can these markers be removed?

decibyte replied:
Yes, of course. Sorry for not cleaning up after myself. I'll update the PR later today.

decibyte commented:
@Lekensteyn I've now removed the stuff we talked about and also rewrapped the text in `devices.md`, which didn't comform to the 80col rule after I added k830 to the document.

Lekensteyn commented:
Looks good, thanks!
